integral_view/templates/format_disks.html

```diff
@@ -1,25 +1,29 @@
-{% extends 'logged_in_base.html' %}
+{% extends 'storage_base.html' %}
```

Comment:
You can only use this screen to format one disk at a time right? If so, this file should not be called format_disks.html, it should be format_disk.html